formats/graphviz/SRF_Graph.php

```diff
@@ -112,7 +115,7 @@ protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		$graphInput .= "}";
 		
 		// Calls graphvizParserHook function from MediaWiki GraphViz extension
-		$result = GraphViz::graphvizParserHook( $graphInput, "", $GLOBALS['wgParser'], "" );
+		$result = $wgParser->recursiveTagParse( "<graphviz>$graphInput</graphviz>" );
```

Comment:
`$GLOBALS['wgParser']` is better in the sense that you do not pollute the whole function scope with the global